packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -92,6 +103,9 @@ export class Accordion extends FASTElement {
 
     private resetItems(): void {
         this.accordionItems.forEach((item: AccordionItem, index: number) => {
+            if (item.expanded === true) {
+                item.removeAttribute("aria-disabled");
```

Comment:
Can you elaborate on the thought behind this interaction? IE, if someone sets expanded, this would no longer be disabled? Is that pattern documented as part of the ARIA specification or somewhere?

Reply:
Hi @chrisdholt thanks for your reply and sorry for not getting back earlier.

Yes, it has been confusing. I'm going to try to explain my assumptions so we can clarify it a bit and you can point me in the right direction.

Initially and driven by the issue https://github.com/microsoft/fast/issues/4160 created by @eljefe223, I thought that the fast-accordion had implemented a behaviour mentioned briefly in the w3c spec that requires a panel to be expanded at all times and it can’t be closed. That matches exactly the issue, where the aria-disabled attribute was missing and that was what I tried to solve in the PR.

You can find more information about it in these links:
- https://w3c.github.io/aria-practices/#accordion
- https://w3c.github.io/aria-practices/examples/accordion/accordion.html

You can check that in the wc3 example, the selected item has got a inner button with an attribute aria-disabled=true and that attribute is missing in the fast accordion item

**Right now I think that the fact that the fast accordion items can't be closed wasn't intentional and it is a bug. It seems the the fast accordion only supports this configuration** 

1. **Multi**: allow multiple panels expanded ( AccordionExpandMode = "multi")
2. **Single**: allow one panel expanded at the time (AccordionExpandMode = "single")

**where all the items can be closed in both modes.**

If that is correct, there is a bug in the fast accordion on single mode because it doesn’t allow to close all the accordion-items. I would be pleasant to create a issue for it to send a pull request to fix it.